index.html

```diff
@@ -193,20 +194,27 @@ <h2 class="conta">CONTACT US</h2>
     if you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
 </p>
```

Comment:
- [ ] The color for the `<h2>` in the Figma design is `#9390FF`.
#### Impl
![image](https://user-images.githubusercontent.com/44057863/177427061-dfd565f4-ad43-4a35-a75a-7d3c8e6ca373.png)
#### Design
![image](https://user-images.githubusercontent.com/44057863/177427129-7104c40f-58b0-42d2-98a2-78d1595dc7fd.png)

- [ ] The font foamily for the heading is `Roboto`. Kindly use that font-family in your implementation.